tests/dbus/test_dbus_client.cpp

```diff
@@ -273,6 +273,28 @@ void CheckEphemeralKey(ThreadApiDBus *aApi)
     TEST_ASSERT(enabled == true);
 }
 
+void CheckBorderAgentInfo(const threadnetwork::TelemetryData_BorderAgentInfo *aBorderAgentInfo)
```

Comment:
nit: const reference 
```suggestion
void CheckBorderAgentInfo(const threadnetwork::TelemetryData_BorderAgentInfo &aBorderAgentInfo)
```

Reply:
Done